addons/packages/vsphere-cpi/1.22.5/bundle/.imgpkg/images.yml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: imgpkg.carvel.dev/v1alpha1
+images:
+- annotations:
+    kbld.carvel.dev/id: gcr.io/cloud-provider-vsphere/cpi/release/manager:v1.22.4
```

Comment:
You need to remove this file and run below command.
```
make lock-package-images PACKAGE=vsphere-cpi VERSION=1.22.5 TAG=1.22.5`
`make push-package PACKAGE=vsphere-cpi VERSION=1.22.5 TAG=1.22.5`
```
It will update the file and push the image